react/LocationSearch.tsx

```diff
@@ -147,12 +142,13 @@ const LocationSearch: React.FC<LocationSearchProps> = ({
       address => address.description === selectedAddress
     )?.externalId
 
-    if (id) {
-      setSearchTerm(selectedAddress)
-      executeGetAddressByExternalId({ variables: { id } })
-    } else {
+    if (id === undefined) {
```

Comment:
why not

```suggestion
    if (id == null) {
```

Reply:
Hmm, linter told me that id could be undefined, so I just googled how to check for undefined and found this: https://stackoverflow.com/a/3390468/7651928

So, why do you think it's a good idea to compare it to null? Or why not just check for falsy values (in this case seems to be ok)?

```suggestion
    if (!id) {
```

Reply:
I had suggested comparing to null (using `==` instead of `===`) because it checks for both `null` and `undefined`.

```js
console.log(null == undefined) // true
```